core/src/main/java/org/opensearch/sql/data/model/ExprDateValue.java

```diff
@@ -22,23 +21,19 @@
 import org.opensearch.sql.data.type.ExprType;
 import org.opensearch.sql.exception.SemanticCheckException;
 
-/**
- * Expression Date Value.
- */
+/** Expression Date Value. */
 @RequiredArgsConstructor
 public class ExprDateValue extends AbstractExprValue {
 
   private final LocalDate date;
 
-  /**
-   * Constructor of ExprDateValue.
-   */
+  /** Constructor of ExprDateValue. */
   public ExprDateValue(String date) {
     try {
       this.date = LocalDate.parse(date, DATE_TIME_FORMATTER_VARIABLE_NANOS_OPTIONAL);
     } catch (DateTimeParseException e) {
-      throw new SemanticCheckException(String.format("date:%s in unsupported format, please use "
-          + "yyyy-MM-dd", date));
+      throw new SemanticCheckException(
+          String.format("date:%s in unsupported format, please use " + "yyyy-MM-dd", date));
```

Comment:
```suggestion
          String.format("date: %s in unsupported format, please use 'yyyy-MM-dd'", date));
```

Reply:
Partly resolved in da35a7eea9faf8e09045a901a03f898a73e2bbd2. (no additional quotes)